FlowerShowerService/Data/Entities/Product.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace FlowerShowerService.Data.Entities
+{
+    public class Product
```

Comment:
Are you suggesting we use one Product entity for both boutiques and single flowers? Maybe it's okey, we probably don't need a way to track boutique's composition. We need description for product tho.

Reply:
I'll add description then.
I think you mean bouquet (not boutique?).